invenio_app_rdm/config.py

```diff
@@ -362,3 +362,26 @@ def _(x):
 }
 
 APP_RDM_RECORDS_EXPORT_URL = "/records/<pid_value>/export/<export_format>"
+
+APP_RDM_DEPOSIT_FORM_DEFAULTS = {
+    "publication_date": lambda: datetime.now().strftime("%Y-%m-%d"),
+    "rights": [
+        {
+            "id": "cc-by-4.0",
+            "title": "Creative Commons Attribution 4.0 International",
```

Comment:
Minor: This data should somehow be retrieved from the license vocabulary. Perhaps via a callable:

```
from invenio_vocabularies.proxies import current_service

def default_license(id_):
    def callable():
         item = current_service.read(('licenses', id_), system_identity).data
         return {"id": item.id, ...}
   return callable

APP_RDM_DEPOSIT_FORM_DEFAULTS = {
     "publication_date": lambda: datetime.now().strftime("%Y-%m-%d"),
     "rights": default_license('cc-by-4.0'),
```